app/components/views/Bond/index.tsx

```diff
@@ -39,6 +39,7 @@ import { RootState, useAppDispatch } from "state";
 import { setBondAllowance } from "state/user";
 import { redeemBond, setBond } from "state/bonds";
 import InfoOutlined from "@mui/icons-material/InfoOutlined";
+import { Box, Checkbox } from "@mui/material";
```

Comment:
Something to get used to, VSCode auto-import will do this, but for better tree-shaking we need to import individually:

```
import Box from "@mui/material/Box";
import Checkbox from "@mui/material/Checkbox";
```

Reply:
updated, I will keep this in mind for the future.